src/pinnable_mapped_file.cpp

```diff
@@ -272,13 +272,13 @@ void pinnable_mapped_file::save_database_file() {
 
       if(time(nullptr) != t) {
          t = time(nullptr);
-         std::cerr << "              " << offset/(_file_mapped_region.get_size()/100) << "% complete..." << std::endl;
+         std::cerr << "CHAINBASE: Writing \"" << _database_name << "\" database file, " << offset/(_file_mapped_region.get_size()/100) << "% complete..." << std::endl;
       }
    }
-   std::cerr << "           Syncing buffers..." << std::endl;
+   std::cerr << "CHAINBASE: Writing \"" << _database_name << "\" database file, flushing buffers..." << std::endl;
    if(_file_mapped_region.flush(0, 0, false) == false)
       std::cerr << "CHAINBASE: ERROR: syncing buffers failed" << std::endl;
```

Comment:
Do you want to change `syncing buffers` to `Writing`?

Reply:
yes true probably should be changed to "flushing" to match the change in wording above it